frontend/assets/javascripts/src/modules/form/stripe.es6

```diff
@@ -0,0 +1,37 @@
+import * as payment from 'src/modules/payment';
+import bean from 'bean'
+import $ from '$'
+export function init() {
+    let handler = window.StripeCheckout.configure(guardian.stripeCheckout);
+    let success = false;
+    const button = $('.js-stripe-checkout');
+    const email = button.data('email');
+    bean.on(window, 'popstate', handler.close);
+
+    const open = (e) => {
+        if (payment.validateForm()) {
+            payment.open();
+            handler.open({
+                description: 'So I could do with some good copy for this, and what it needs to contain like price etc.',
+                panelLabel: 'Approved button copy!',
+                email: email,
```

Comment:
Oh, so to follow on from https://github.com/guardian/membership-frontend/pull/1436/files#r102215893, after discussion with @Aware it becomes clear that if we don't send an `email` to Stripe, Stripe will require the user to enter their email on the overlay (something we'd obviously rather avoid).